src/ray/object_manager/common.h

```diff
@@ -39,8 +39,8 @@ using RestoreSpilledObjectCallback =
 /// A struct that includes info about the object.
 struct ObjectInfo {
   ObjectID object_id;
-  int64_t data_size;
-  int64_t metadata_size;
+  int64_t data_size = 0;
+  int64_t metadata_size = 0;
```

Comment:
These initializations are used to avoid the failure from UBSAN, the error message is:
```bash


[ RUN      ] ObjectLifecycleManagerTest.RemoveReferenceTwoRef
--
  | bazel-out/k8-opt/bin/_virtual_includes/plasma_store_server_lib/ray/object_manager/common.h:53:54: runtime error: signed integer overflow: 5921777712514757477 + 8315161630257148773 cannot be represented in type 'long int'
```

The cause is these 2 fields are not initialized and then the result of `data_size + metadata_size` is overflow.

Why it doesn't happen on master?
Because this PR may change the memory view of the process. I tested master, `data_size` and `metadata_size` are also big numbers but the sum of them is a little smaller than the max value of `int64_t`.

Reply:
Nice catch. cc @scv119 can you check setting these to 0 & 0 is reasonable?